packages/cli/src/commands/hydrogen/build.ts

```diff
@@ -19,7 +19,7 @@ const LOG_WORKER_BUILT = '📦 Worker built';
 
 export default class Build extends Command {
   static description = 'Builds a Hydrogen storefront for production.';
-  static flags = {
+  static flags: any = {
```

Comment:
Was this necessary to add on all of the commands?

Reply:
I was getting type errors after bumping oclif/core to 2.8.7 (needed to match the version used by oxygen-cli) that I could not resolve in any other way.